aries-site/src/tests/utils.js

```diff
@@ -30,7 +30,7 @@ export const getTabCount = ClientFunction(expectedPath => {
   let hrefIndex;
   for (let i = 0; i < tabbableElements.length; i += 1) {
     if (tabbableElements[i].getAttribute('href') === expectedPath) {
-      hrefIndex = i;
+      hrefIndex = i + 1;
```

Comment:
What was the reasoning behind having the `index + 1` ?

Reply:
I believe previously when we had the tabbable box around the search input it was working, but logically off by one in terms of how a user would tab. The reason you add one is because array indexes start at 0. So, if the item I want to tab to is the first item in the array (index = 0), then I'd want to tab 1 time to get there (index + 1) -- maybe it's make more sense for the function to return hrefIndex + 1 instead of reassigning it there. I'll update and add a clarifying comment for why this is the case.

Reply:
sounds good thank you! everything else looks good!